applications/DG-Max/Algorithms/DGMaxEigenvalue.cpp

```diff
@@ -162,8 +164,12 @@ class DGMaxEigenvalue<DIM>::SolverWorkspace {
     /// Shell matrix P*product_, where P is the projection operator
     Mat shell_;
 
-    // Solver
-    EPS solver_;
+    // SLEPC Solver
+    EPS eps_solver_;
```

Comment:
Could you rename these to `epsSolver_` and `jdmaxSolver_` to follow the conventions?